src/System.Text.Json/tests/JsonElementWithNodeParentTests.cs

```diff
@@ -122,7 +122,10 @@ public static void TestString()
         [Fact]
         public static void TestBytesFromBase64()
         {
-            Assert.Throws<NotSupportedException>(() => new JsonString().AsJsonElement().GetBytesFromBase64());
+            Assert.Equal(Encoding.UTF8.GetBytes("value"), new JsonString("dmFsdWU=").AsJsonElement().GetBytesFromBase64());
+            Assert.Throws<FormatException>(() => new JsonString("Not base-64").AsJsonElement().GetBytesFromBase64());
```

Comment:
Add `TryGetBytesFromBase64` tests where these should be returning false

Reply:
Done!